src/kirby/components/toolbar/toolbar.component.scss

```diff
@@ -0,0 +1,33 @@
+@import '../../scss/web-imports';
+
+.toolbar {
+  --background:  #{get-color('background-color')};
+  --color: #{get-color('background-color-contrast')};
+}
+
+@each $color-name, $color-value in $main-colors {
+  :host(.#{$color-name}) {
+    ion-toolbar {
+      --background:  #{get-color($color-name)};
+      --color: #{get-color($color-name+'-contrast')};
+    }
+  }
+}
+
+.toolbar-item {
```

Comment:
This is my own opinion and we have not talked(?) about this in kirby in general.

But remember we use angular and have scoped styling. So there is no need for double naming of classes. I know by the file name that this is the styling for the toolbar. So i find naming the class `item` would give me the same info.

But again this is just nitpicking from my side 😄